content/events/2021/04/2021-04-28-federal-developmental-opportunities-workshop.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Federal Developmental Opportunities Workshop
+deck: Find out about developmental opportunities available for federal
+  government employees.
+kicker: Professional Development
+summary: The Federal Leadership and Professional Development Seminar Series
+  (FLPDSS) is hosting a cross-agency workshop focusing on developmental
+  opportunities for federal employees.
+host: Federal Leadership and Professional Development Community of Practice
+event_organizer: Digital.gov
+registration_url: https://cma.ahrq.gov/cma/welcome.jsp?code=may2021
+date: 2021-05-04 09:00:00 -0500
+end_date: 2021-05-04 15:00:00 -0500
+# See all topics at https://digital.gov/topics
+topics:
+  - professional-development
+# See all authors at https://digital.gov/authors
+authors:
+  - kim-m-wittenberg
+slug: federal-developmental-opportunities-workshop
+# zoom, youtube_live, adobe_connect, google
+event_platform: zoom
+---
+The Federal Leadership and Professional Development Seminar Series (FLPDSS) will host a cross-agency workshop focusing on developmental opportunities for federal employees. The workshop will provide information on how employees can develop themselves, and will provide insight in how supervisors develop their teams, and bring in outside talent.
+
+Presentations will inform attendees on opportunity options including: coaching, mentoring, details to other federal agencies, formal federal experiential programs, and [USAJOBS Open Opportunities](https://openopps.usajobs.gov/). The workshop will end with a panel discussion on successful strategies for starting, sustaining, and evaluating formal mentoring programs.
+
+## Agenda
+
+### Morning Session (09:00 am - 12:10 pm, ET)
+
+* **9:00 am - 10:15 am:** Developmental Opportunities – What they are, why they’re valuable, and how federal employees can host and receive them
```

Comment:
Add 2 spaces to the end of this line to force the next sentence to a new line. So that the speakers are listed underneath. This should be done for each session.

Reply:
@saracope is the period at the end of the line normal? When I hit the space bar twice it auto puts a period at the end of the line. I'm assuming I should just delete the period? or add the <br> then double space / return?